src/js/candidate-detail.js

```diff
@@ -0,0 +1,19 @@
+let urlParams = new URLSearchParams(window.location.search);
+let slug = urlParams.get('slug') || window.location.pathname.split('/')[2];
+// console.log(slug);
+// console.log(window.location.pathname);
+
+let html = document.getElementById('mustache-template').innerHTML;
+let htmlContainer = 'pageContent';
+
+module.exports = (core) => {
+  core.getResource('candidates', { slug }).then((candidate) => {
+    console.log(candidate.data[0]);
+
+    candidate.data[0].name_full = `${candidate.data[0].name_first}  ${candidate.data[0].name_last}`;
+
+    let output = core.Mustache.render(html, { candidate: candidate.data[0] });
```

Comment:
This is basically how we did the templating with the first frontend too. I'm ok with doing it this way 👍 what are your thoughts @crsmyz?

Reply:
Ah, I hadn't realized that.

Reply:
I think its fine, especially if its just a poc